pkgs/development/python-modules/anthropic/default.nix

```diff
@@ -87,11 +90,14 @@ buildPythonPackage rec {
     "ignore::DeprecationWarning"
   ];
 
-  meta = with lib; {
```

Comment:
Removing the `with lib` here but then using `with lib.maintainers` below seems a bit counterintuitive, but it’s fine I guess.

Reply:
It does feel wierd, yet it seems to be a common patter: ~2400 instances of `lib.maintainers.` v. 9400 instances of `lib.maintainers;`. I normally drop the `with` for a single maintainer entry but not multiples.